content/getting-started/usage.md

```diff
@@ -233,11 +234,11 @@ By using Hugo's server in production, you are able to deploy just the source fil
 
 Interested? Here are some great tutorials contributed by Hugo users:
 
-* [hugo, syncthing](http://fredix.xyz/2014/10/hugo-syncthing/) (French) by Frédéric Logier (@fredix)
+* [hugo, syncthing](https://fredix.xyz/2014/10/hugo-syncthing/) (French) by Frédéric Logier (@fredix)
```

Comment:
Right, but this is about using Hugo's server *in production*. The french tutorial is 2.5 years old, so I removed it (not currently on the live site). I don't think it adds a lot of value...